patches/api/0423-Paper-Item-API.patch

```diff
@@ -0,0 +1,653 @@
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/property/value/SetValue.java
+@@ -0,0 +1,12 @@
++package io.papermc.paper.property.value;
++
++import java.util.Set;
++
++public interface SetValue<V> extends Value<Set<V>> {
++
++    boolean contains(V value);
++
++    void add(V value);
++
++    void remove(V value);
++}
+diff --git a/src/main/java/io/papermc/paper/property/value/Value.java b/src/main/java/io/papermc/paper/property/value/Value.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..fab6096d1e0edece524f62bcf09990c3ab897b2d
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/property/value/Value.java
+@@ -0,0 +1,12 @@
++package io.papermc.paper.property.value;
++
++import io.papermc.paper.property.PropertyKey;
++import org.jetbrains.annotations.Nullable;
++
++public interface Value<T> {
++
++    @Nullable
++    T getValue();
++
++    PropertyKey<? extends Value<T>> getKey();
++}
+diff --git a/src/main/java/io/papermc/paper/property/value/handler/PropertyValueHandler.java b/src/main/java/io/papermc/paper/property/value/handler/PropertyValueHandler.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..46fb5f54feeefc7cf954bec257bff0a682fae3ff
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/property/value/handler/PropertyValueHandler.java
+@@ -0,0 +1,4 @@
++package io.papermc.paper.property.value.handler;
++
++public interface PropertyValueHandler {
++}
+diff --git a/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java b/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
+index cb3c25ef46b279ffdde87f47f729eb8aa7549c1c..5247daad3356d23c193f8fc93225c98ed6f201c4 100644
+--- a/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
++++ b/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
+@@ -2,6 +2,12 @@ package org.bukkit.plugin.java;
+ 
+ import com.google.common.base.Preconditions;
+ import com.google.common.io.ByteStreams;
++import org.bukkit.plugin.InvalidPluginException;
++import org.bukkit.plugin.PluginDescriptionFile;
++import org.bukkit.plugin.SimplePluginManager;
++import org.jetbrains.annotations.NotNull;
++import org.jetbrains.annotations.Nullable;
++
+ import java.io.File;
+ import java.io.IOException;
+ import java.io.InputStream;
+@@ -20,11 +26,6 @@ import java.util.jar.JarEntry;
+ import java.util.jar.JarFile;
+ import java.util.jar.Manifest;
+ import java.util.logging.Level;
+-import org.bukkit.plugin.InvalidPluginException;
+-import org.bukkit.plugin.PluginDescriptionFile;
+-import org.bukkit.plugin.SimplePluginManager;
+-import org.jetbrains.annotations.NotNull;
+-import org.jetbrains.annotations.Nullable;
+ 
+ /**
+  * A ClassLoader for plugins, to allow shared classes across multiple plugins
+@@ -66,6 +67,24 @@ public final class PluginClassLoader extends URLClassLoader { // Spigot
+         this.logger = com.destroystokyo.paper.utils.PaperPluginLogger.getLogger(description); // Paper - Register logger early
+ 
+         try {
++
```

Comment:
what is this lol?

Reply:
Guess that is not intended as the only changes here are new whitespace and a import organize.